examples/js/api-lifecycle.test.js

```diff
@@ -0,0 +1,173 @@
+'use strict'
+
+const shared = require('./shared')
+const request = require('supertest')(shared.apiUrl)
+
+const axios = require('axios')
+const { describe, expect } = require('@jest/globals')
+const api = axios.create({
+    baseURL: shared.apiUrl,
+    timeout: 1000000000
```

Comment:
I would set a reasonable timeout here, or we're suggesting to our customers that our product performance is at issue.